content/en/docs/ops/configuration/traffic-management/dns-proxy/test.sh

```diff
@@ -29,7 +29,7 @@ snip_dns_capture_in_action_2
 
 # configure service entries and verify
 snip_dns_capture_in_action_1
-_verify_first_line snip_dns_capture_in_action_3 "$snip_dns_capture_in_action_3_out"
+_verify_contains snip_dns_capture_in_action_3 "$snip_dns_capture_in_action_3_out"
```

Comment:
You should probably change the index.md text section to include the two lines of expected output, since that's what the user should see now? `verify_contains` should work even if the compare string is multi-line.

Reply:
Done @frankbu